vms/platformvm/state/state.go

```diff
@@ -214,7 +214,13 @@ type State interface {
 
 	SetHeight(height uint64)
 
-	GetCurrentValidatorSet(ctx context.Context, subnetID ids.ID) (map[ids.ID]*validators.GetCurrentValidatorOutput, uint64, error)
+	// GetCurrentValidators returns subnet and L1 validators for the given
```

Comment:
This comment might be more appropriate on top of the function implementation than for the interface.

Reply:
let's keep it same with other interface function comments